src/views/AddExpenses.js

```diff
@@ -1,76 +1,100 @@
 import { useState } from 'react';
-import { convertEurosToCents } from '../utils/helpers';
+import { convertEurosToCents, getUKFormattedDate } from '../utils/helpers';
 
-const AddExpenses = ({ addExpense }) => {
+import ConfirmMonthModal from '../components/ConfirmMonthModal';
+
+const AddExpenses = ({ addExpense, selectedMonth, setSelectedMonth }) => {
   const [title, setTitle] = useState('');
   const [date, setDate] = useState('');
   const [amount, setAmount] = useState('');
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
i was briefly confused which modal is meant here...

Reply:
renamed!